bingo/symbolic_regression/agraph/mutation.py

```diff
@@ -233,3 +249,172 @@ def _get_random_prune_location(individual):
             return None
         index = np.random.randint(len(indices))
         return indices[index]
+
+    def _fork_mutation(self, individual):
```

Comment:
The new code for this looks solid

I think you are right that it could benefit from a bit of documentation in the next PR though

Reply:
Thanks, documentation will be added in the next commits that are pushed.